{"Launchpad":"https://bugs.launchpad.net/bugs/1433204","Id":"1433204","Gerrit":"https://review.openstack.org/165560","IdGerrit":"165560","Files":["nova/tests/unit/virt/libvirt/test_volume.py","nova/virt/libvirt/volume.py"],"Lines_Inseted":3,"Lines_Deleted":9,"CommitParent":"d731901f06f61413f222395b12cac3c3b3e425de","Classification":"Bug","KeywordsTitle":"Cannot recognize warning message","Description":"multipath -l\" command may report an error, PARSER cannot recognize this warning messag\nvolume cannot be attached to instances correctly.","KeywordsCommit":" Remove error messages from multipath, This fixes an issue in _get_multipath_device_name() ","Comments":"Faulty feature"}